.eslintrc.base.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Might think about this in the next update. I don’t want to put the config into a separate package yet, because this is beyond the scope of the current task.
Updates in this PR only fix the problem with eslint for demo pages